shared/controller-api/src/main/swagger/Controller.yaml

```diff
@@ -388,6 +393,196 @@ paths:
           description: Scope or reader group with given name not found
         500:
           description: Internal server error while fetching reader group details
+  /health:
+    get:
+      tags:
+        - "Health"
+      operationId: getHealth
+      description: Return the Health of the Controller service.
+      produces:
+        - application/json
+      responses:
+        200:
+          description: The Health result of the Controller.
+          schema:
+            $ref: "#/definitions/HealthResult"
+        500:
+          description: Internal server error while fetching the Health.
+  /health/readiness:
+    get:
+      tags:
+        - "Health"
+      operationId: getReadiness
+      description: Fetch the ready state of the Controller service.
+      produces:
+        - application/json
+      responses:
+        200:
+          description: The ready status.
+          schema:
+            type: boolean
+        500:
+          description: Internal server error while fetching the ready state of the Controller.
+  /health/liveness:
+    get:
+      tags:
+        - "Health"
+      operationId: getLiveness
+      description: Fetch the liveness state of the Controller service.
+      produces:
+        - application/json
+      responses:
+        200:
+          description: The alive status.
+          schema:
+            type: boolean
+        500:
+          description: Internal server error while fetching the liveness state of the Controller.
+  /health/details:
```

Comment:
What is the difference between "/health/details" API and the "/health" API? Both seem to return the _Health_ object for Controller.

Reply:
`/health/details` returns just the `Map<String, Object>` associated with the `Health` object, instead of the `Health` object itself.

Reply:
Okay.